src/components/main/HomeScreen.js

```diff
@@ -234,7 +234,6 @@ const styles = StyleSheet.create({
     alignItems: "center",
     height: Math.round(scale(height)),
     width: width,
-    backgroundColor: COLORS.black,
```

Comment:
This needs to stay. Even though it doesn't show up at all, occasionally you get a brief white flash between when the splash screen dissappears and the background loads in. The black background is like the canvas behind the background image that makes it unnoticeable.